core/src/main/java/io/questdb/griffin/engine/functions/columns/BinColumn.java

```diff
@@ -57,6 +57,11 @@ public long getBinLen(Record rec) {
         return rec.getBinLen(columnIndex);
     }
 
+    @Override
+    public boolean isReadoutStateless() {
+        return true;
```

Comment:
Why, I'd think it will go down to `MemoryCR.ByteSequenceView` which is not thread safe
```suggestion
        return false;
```

Reply:
done